### ISSUE TITLE
Generated XML documentation has error

### ISSUE THREAD

**Issue by zlepper** [rebus-org/Rebus#1004]
Whenever we build and use inheritdoc to copy documentation so it works with swagger, we get this warning:
```
C:\Users\rhdh\.nuget\packages\rebus\6.6.1\lib\netstandard2.1\Rebus.xml(6376,1): InheritDocTask warning IDT001: XML parse error in referenced doc: Data at the root level is invalid. Line 6376, position 1. [C:\dev\digizuite.core\Apps\DSL-service\DSL-service.csproj]
```

This seems to be an issue that is specific to Rebus 6.6.1. The 7.0 beta doesn't seem to have the same issue. 

We use this (in the csproj) to trigger the inheritdoc:

```xml

  <PropertyGroup>
    <GenerateDocumentationFile>true</GenerateDocumentationFile>
    <InheritDocEnabled>true</InheritDocEnabled>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="SauceControl.InheritDoc" Version="1.3.0">
      <PrivateAssets>all</PrivateAssets>
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
    </PackageReference>
  </ItemGroup>
```
It doesn't block our build or anything, it's just an annoying warning :)

**Comment by mookid8000** [comment]
Whoa, that's weird! Check out what the XML looks like towards the end:
```xml
        <member name="T:FastExpressionCompiler.ExpressionCompiler.EmittingVisitor">
            <summary>Supports emitting of selected expressions, e.g. lambdaExpr are not supported yet.
            When emitter find not supported expression it will return false from <see cref="M:FastExpressionCompiler.ExpressionCompiler.EmittingVisitor.TryEmit(System.Linq.Expressions.Expression,System.Collections.Generic.IList{System.Linq.Expressions.ParameterExpression},System.Reflection.Emit.ILGenerator,FastExpressionCompiler.ExpressionCompiler.ClosureInfo)"/>, so I could fallback
            to normal and slow Expression.Compile.</summary>
        </member>
    </members>
</doc>
,System.Collections.Generic.IList{System.Linq.Expressions.ParameterExpression},System.Reflection.Emit.ILGenerator,FastExpressionCompiler.ExpressionCompiler.ClosureInfo)"/>, so I could fallback
            to normal and slow Expression.Compile.</summary>
        </member>
    </members>
</doc>

```

It's as if the docs XML has been written without completely overwriting the previous contents.

I'll try to be alert of whether this happens again on the next release. Also, I've (re-)released Rebus 6.6.1 as 6.6.2, which should be available on NuGet.org in a few minutes.

Thanks for letting me know 🙂

**Comment by zlepper** [comment]
Waves wand of necromancy:
![image](https://user-images.githubusercontent.com/1499810/157248514-12ec915a-6170-46ad-947b-66fdccf6f253.png)

I could have sworn I double checked the 6.6.2 release when you published that, but apparently this is still an issue, and seems even more broken than before? @mookid8000 ?

**Comment by mookid8000** [comment]
I think this is fixed in Rebus 6.6.4!

My hypothesis is that the corruption happened because of a combination of having multiple build targets and having customized the output paths for the generated XML files.

I've removed the customized paths now, and I think the XML files look intact.
